packages/gatsby-source-mongodb/src/__tests__/stringify-object-ids.js

```diff
@@ -0,0 +1,168 @@
+const stringifyObjectIds = require(`../stringify-object-ids`)
+const ObjectID = require(`mongodb`).ObjectID
+
+test(`it is a function`, () => {
+  expect(stringifyObjectIds).toEqual(expect.any(Function))
+})
+
+test(`ObjectIDs are instances of mongodb.ObjectID and not strings`, () => {
+  const objectId = new ObjectID(`5c6d85cff2bc0b7a5da61124`)
+  expect(objectId instanceof ObjectID).toBeTruthy()
+  expect(typeof objectId == `string`).toBeFalsy()
+})
+
+test(`ObjectIDs are not Equal to Strings with the same value`, () => {
+  const objectId = new ObjectID(`5c6d85cff2bc0b7a5da61124`)
+  const str = `5c6d85cff2bc0b7a5da61124`
+  expect(objectId).not.toEqual(str)
+  expect(objectId)
+})
+
+test(`it returns stringified object id when passed one`, () => {
+  const objectId = new ObjectID(`5c6d85cff2bc0b7a5da61124`)
+  expect(stringifyObjectIds(objectId)).toEqual(`5c6d85cff2bc0b7a5da61124`)
+})
+
+test(`when passed an array of ObjectIDs it returns an array of strings`, () => {
+  const str1 = `5c6d85cff2bc0b7a5da61124`,
+    str2 = `5c6d85fdf2bc0b7a5da6112a`,
+    str3 = `5c6d863df2bc0b7a5da61132`,
+    obj1 = new ObjectID(str1),
+    obj2 = new ObjectID(str2),
+    obj3 = new ObjectID(str3),
+    arr = [obj1, obj2, obj3],
+    res = [str1, str2, str3],
+    stringifiedArr = stringifyObjectIds(arr)
+
+  expect(stringifiedArr[0]).toEqual(res[0])
+  expect(stringifiedArr[1]).toEqual(res[1])
+  expect(stringifiedArr[2]).toEqual(res[2])
+})
+
+test(`when passed an array of object literals with ObjectIDs it returns an array of object literals with strings`, () => {
+  const str1 = `5c6d85cff2bc0b7a5da61124`,
```

Comment:
Same idea here--I think this is a bit manual. If we can re-write it I think that would be preferable, e.g.

```
const ids = [
  // id1
  // id2
  // id3
]

expect(
  stringifyObjectIds(
    ids.map((id, index) => ({
      key: index,
      id: new ObjectID(id)
    })
  )
    .map(obj => obj.id)
).toEqual(ids)
```